fuzz/Cargo.toml

```diff
@@ -7,7 +7,7 @@ publish = false
 
 [dependencies]
 tinyvec = { path = "..", features = ["alloc", "nightly_slice_partition_dedup"] }
-arbitrary-model-tests = { git = "https://github.com/jakubadamw/arbitrary-model-tests" }
+rutenspitz = "0.2"
```

Comment:
sup

Reply:
Fuzzer needs to be fixed, because currently it doesn't even pass a `cargo check`.
It is not urgent, because CI doesn't check it, but it would be good if I wasn't the only one catching bugs in tinyvec ;)

Reply:
This change in Cargo.toml was needed, because it broke `cargo build` in tinyvec